tests/e2e/operator/operator_test.go

```diff
@@ -99,14 +97,29 @@ var _ = Describe("Operator", Ordered, func() {
 		})
 
 		It("deploys all the CRDs", func() {
-			Eventually(kubectl.GetCRDs).
-				Should(ContainElements(crds), "Istio CRDs are not present; expected list to contain all elements")
+			Eventually(func(g Gomega) {
+				result, err := k8sclientextension.ApiextensionsV1().CustomResourceDefinitions().List(context.TODO(), metav1.ListOptions{})
+				g.Expect(err).ToNot(HaveOccurred(), "Error getting CRDs")
+				var crdNames []string
+				for _, crd := range result.Items {
+					crdNames = append(crdNames, crd.ObjectMeta.Name)
+				}
+				g.Expect(crdNames).To(ContainElements(crds), "Istio CRDs are not present; expected list to contain all elements")
+			}).Should(Succeed(), "Istio CRDs are not present")
```

Comment:
Make all the get and check of the crds in the code is more extensive in the code (All this was done before in the func kubectl xxx), but I think is clear what is done. btw, I added the `g.Expect(err).ToNot(HaveOccurred(), "Error getting CRDs")` because I can't wrap inside any other way to check that non error ocurred